elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/ElideAsyncConfiguration.java

```diff
@@ -72,4 +70,10 @@ public AsyncCleanerService buildAsyncCleanerService(Elide elide, ElideConfigProp
     public AsyncQueryDAO buildAsyncQueryDAO(Elide elide) {
         return new DefaultAsyncQueryDAO(elide, elide.getDataStore());
     }
+
+    @Override
+    public void afterPropertiesSet() {
+        ExecuteQueryHook executeQueryHook = new ExecuteQueryHook(asyncExecutorService);
```

Comment:
I thought we were going to do all of this logic in the AsyncExecutorService bean?